src/backend/vector/avx2/edwards.rs

```diff
@@ -188,7 +188,7 @@ impl From<ExtendedPoint> for CachedPoint {
         let mut x = P.0;
 
         x = x.blend(x.diff_sum(), Lanes::AB);
-        // x = (X1 - Y1, X2 + Y2, Z2, T2) = (S2 S3 Z2 T2)
+        // x = (Y2 - X2, Y2 + X2, Z2, T2) = (S2 S3 Z2 T2)
```

Comment:
this was just a comment issue or the logic actually changed?

Reply:
The logic did not change ( https://github.com/dalek-cryptography/curve25519-dalek/commit/8da05f7e90b59b6dea6f72c4d4cad238647b1b60#diff-f507d38947f74ad1450f16fa341b60ddb21d9d77b7db0cb700fac4fa3c783eeb )